build.cake

```diff
@@ -292,36 +293,41 @@ Task("Prepare-Release")
 Task("Update-Version")
   .Does(() =>
 {
-  if(string.IsNullOrWhiteSpace(version)) {
-    throw new CakeException("No version specified!");
-  }
+    if(string.IsNullOrWhiteSpace(version)) {
+        throw new CakeException("No version specified!");
+    }
 
-  UpdateProjectJsonVersion(version, projectJsonFiles);
+    UpdateProjectJsonVersion(version, projectJsonFiles);
 });
 
 ///////////////////////////////////////////////////////////////
 
 public void UpdateProjectJsonVersion(string version, FilePathCollection filePaths)
 {
-  Verbose(logAction => logAction("Setting version to {0}", version));
-  foreach (var file in filePaths)
-  {
-    var project = System.IO.File.ReadAllText(file.FullPath, Encoding.UTF8);
+    Verbose(logAction => logAction("Setting version to {0}", version));
 
-    project = System.Text.RegularExpressions.Regex.Replace(project, "(\"version\":\\s*)\".+\"", "$1\"" + version + "\"");
+    foreach (var file in filePaths)
+    {
+        var project =
+            System.IO.File.ReadAllText(file.FullPath, Encoding.UTF8);
 
-    System.IO.File.WriteAllText(file.FullPath, project, Encoding.UTF8);
-  }
-}
+        var projectVersion =
+            new System.Text.RegularExpressions.Regex("(\"version\":\\s*)\".+\"");

```

Comment:
Yes? I have a habit of running code that I change, before I pull-request it 😄 I've not modified the regex